pkg/client/sc/websocket.go

```diff
@@ -0,0 +1,31 @@
+package sc
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"github.com/apache/servicecomb-service-center/pkg/backoff"
+	"github.com/gorilla/websocket"
+)
+
+func (c *LBClient) WebsocketDial(ctx context.Context, api string, headers http.Header) (conn *websocket.Conn, err error) {
+	err = backoff.DelayIn(c.Retries, func() (rerr error) {
```

Comment:
please fetch the latest code and fix the delay problem

Reply:
the backoff isn't expected here, just use for loop to retry the next endpoint as soon as possible

Reply:
Modified, please review